### DISCUSSION TITLE
An introduction to overrides in synced patterns

### DISCUSSION THREAD

**Discussion by ndiego** [WordPress/developer-blog-content#267] [Topic Ideas]
This article will provide an introduction to [overrides in synced patterns](https://github.com/WordPress/gutenberg/issues/59819#top), a new feature coming in WordPress 6.6. The article would cover the following: 

- Explain what synced pattern overrides are and demonstrate how they work from a user perspective
- Explain how they work under the hood and their connection to the Block Bindings API
- Show how to create overrides with a couple examples
- A brief overview of what is on the roadmap for overrides in 6.7+ and links to relevant resources

_The actual article title needs some work._

**Comment by bph** [comment]
This topic has been approved. Please continue the conversation on the issue

- #267

**Comment by ndiego** [comment]
Post has been published: https://developer.wordpress.org/news/2024/06/18/an-introduction-to-overrides-in-synced-patterns/
